linode/manager
# test: [M3-6632] - Add Machine Image region tests

## Description 📝
This adds a handful of region tests (end-to-end) for Machine Image operations:

- Upload an Image via `/images/create/upload`
- Capture an Image from a Linode via `/images/create/disk`
- Update an Image's label and description from landing page
- Delete an Image from landing page

## How to test 🧪
- Rely on automated Cypress tests to ensure that existing Image tests are not impacted
- Run the new tests with this command (takes approx. :
  ```bash
  CY_TEST_SUITE=region CY_TEST_REGION=us-east yarn cy:run -s "cypress/e2e/region/images/*.spec.ts"
  ```
(It takes roughly 5 minutes to run all of the tests against 1 region on my machine)
